packages/storybook-tests/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@nl-design-system-candidate/storybook-tests",
```

Comment:
Ik ben zelf geneigd om dingen zoveel mogelijk in singular te schrijven, omdat er anders veel willekeur en onvoorspelbaarheid is. Hier dan `storybook-test` ipv `storybook-tests`